packages/swagger-editor/src/components/ResetStyles.vue

```diff
@@ -79,4 +80,8 @@ onMounted(() => {
     font-family: var(--theme-font, var(--default-theme-font));
   }
 }
+body.light-mode,
+body.dark-mode {
+  margin: 0;
+}
```

Comment:
One of those classes is applied anyway, can’t we just use `body { margin: 0; }` then?